scripts/EGAP/files_to_import_structure.py

```diff
@@ -0,0 +1,55 @@
+import os
+import shutil
+import argparse
+from distutils.dir_util import copy_tree
+
+get_item_id = lambda _path: _path.split(os.sep)[-1].split(' ')[0]
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-source', '--source', help='')
+parser.add_argument('-destination', '--destination', help='')
+
+
+def main(origin_path, new_path):
+    # Copy whole tree, then pick out anonymous
```

Comment:
I'm presuming source/origin_path here is the raw files from EGAP, and the destination/new_path is the generated directories from the spreadsheet data. If there's only one directory in the destination and 1500 directories in the source, is this going to go through 1500 directories worth of files?

Reply:
I don't really understand this comment. Source and destination have the same number of directories in them one for each project. I have to copy the whole tree under the source (a directory like `20121001AA Findley`) because there's potentially a whole file hierarchy inside it, as per our discussion about supplemental materials. 

Once the tree is copied from inside `20121001AA Findley` to `20121001AA/data/nonanonymous` we walkthourgh all newly copied files for ones that have the anonymous naming conventions, create an `anonymous` folder and move those files in.

Reply:
Is `destination` the directory structure that was made by the Corbin's script? Because, if so, we are going to run that in multiple rounds, and some rounds are only going to have a small number of directories to process, while the files source is going to contain everything from every migration ever. It's not going to shrink, just grow.